### DISCUSSION TITLE
Password forgotten

### DISCUSSION THREAD

**Discussion by XTorX** [mckaywrigley/chatbot-ui#1473] [Help]
Reset doesn't work right!? :(
Thanks

**Comment by spammenotinoz** [comment]
Works for myself when initiated from the front-end, not when initiated from the database.
You need to update your front end-url in your database config though.
Remember to set your Site URL in SupaBase to be your front-end URL or the emailed link won't work.

**Comment by mhalavanja** [comment]
I was not getting the email so I deleted the user from database and created the same one on sign up.
